pkg/graph/schema.resolvers.go

```diff
@@ -1585,6 +1586,7 @@ func (r *systemIntakeResolver) Actions(ctx context.Context, obj *models.SystemIn
 				Name:  action.ActorName,
 				Email: action.ActorEmail.String(),
 			},
+			SystemIntake:           obj,
```

Comment:
Was this really just totally missing before? So querying `action.SystemIntake` would've just died?

Reply:
If so -- good add, but bummer that we missed this till now!

Reply:
[Yeah, as far as I can tell.](https://github.com/CMS-Enterprise/easi-app/blob/main/pkg/graph/schema.resolvers.go#L1588)